kaldb/src/main/java/com/slack/kaldb/logstore/search/SearchQuery.java

```diff
@@ -43,6 +48,8 @@ public String toString() {
         + howMany
         + ", bucketCount="
         + bucketCount
+        + ", chunkIds="
+        + chunkIds
```

Comment:
does this work? i.e chunkIds is a List and just saying `chunkIds` here will it override correctly?

Reply:
Done.